README.md

```diff
@@ -3,3 +3,15 @@ The backend for eduBoard, written in Go with [httprouter](https://github.com/jul
 
 ## Installing
 - Run `dep eunsure` to install dependencies
+
+## Endpoints
+
+- `/` reservered for frontend
+
+### User
+- `/api/register` Register a new user.
+- `api/login` Login an existing user.
+
+### Courses
+- `/api/courses` Get all accessible courses
+- `/api/courses/mine` Get users courses 
```

Comment:
Maybe we actually have to think of something different here. From httprouters readme: 
> Note: Since this router has only explicit matches, you can not register static routes and parameters for the same path segment. For example you can not register the patterns /user/new and /user/:user for the same request method at the same time. The routing of different request methods is independent from each other.

Since the RESTful approach would be to also allow for `/api/courses/:courseId` to get a certain course, this is a conflict.